tests/engine/test_engine.py

```diff
@@ -23,15 +24,17 @@ def mock_rest_configuration(monkeypatch):
         ("my-little-subnet", "whole-golem", "whole-golem"),
     ],
 )
-def test_set_subnet_tag(default_subnet, subnet_arg, expected_subnet, monkeypatch):
+def test_set_subnet_tag(
+    default_subnet, subnet_arg, expected_subnet, monkeypatch, golem_factory, api_config_factory
```

Comment:
again, possibly this should use `mock.Mock` as the `ApiConfig`  ... unless there's a good reason not to...